src/providers/wfs/qgswfsprovider.h

```diff
@@ -137,7 +137,22 @@ class QgsWFSProvider : public QgsVectorDataProvider
      * Get some metadata description of the provider.
      * @return The provider metadata
      */
-    virtual QgsStringMap metadata();
+    virtual QVariantMap metadata();
```

Comment:
Shouldn't these methods override the one in the base class?

    virtual QVariantMap metadata() const override;